aptos-move/framework/aptos-framework/sources/configs/staking_config.move

```diff
@@ -48,13 +55,19 @@ module aptos_framework::staking_config {
             error::invalid_argument(EINVALID_REWARDS_RATE),
         );
 
+        assert!(
+            voting_power_increase_limit > 0 && voting_power_increase_limit <= 100,
```

Comment:
I don't think the upper bound should be 100, we can only tolerate 50% at most without taking down the network even the previous one are all good. 1/(1+0.5) = 2/3

in reality this probably should be less than 10%

Reply:
50 sounds like a more reasonable upper bound. Whether we'll set it to 10% or less in reality shouldn't affect the hardcoded upperbound value here.